packages/libmatio/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/tbeu/matio"
+TERMUX_PKG_DESCRIPTION="A C library for reading and writing Matlab MAT files"
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_LICENSE_FILE="COPYING"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.5.23"
+TERMUX_PKG_SRCURL="https://github.com/tbeu/matio/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=5f25c53e698d06b3119e3a4d0536e1da89983dbb7b0a845270663be02ec95978
+TERMUX_PKG_BUILD_DEPENDS="cmake"
```

Comment:
This line is not needed, cmake is installed by build system automatically. `TERMUX_PKG_BUILD_DEPENDS` means that it downloads cmake that has been cross-compiled for termux, and that cmake cannot be used in any case